features/autonomous-custom-elements.yml

```diff
@@ -0,0 +1,11 @@
+name: Autonomous custom elements
+description: Autonomous custom elements are HTML elements with behavior that you define.
```

Comment:
This description should say something about how to create these elements, tag names with a dash in them.

Reply:
How about….

```suggestion
description: Autonomous custom elements are HTML elements with a hyphenated tag name (like `<example-element>`) that have a behavior you define.
```

Reply:
Look good to me! An example tag name that's shorter and also realistic might be nice, but I struggle to find something that's obviously an example and not defining the scope of the feature. `<login-button>` and `<video-player>` aren't as good as `<example-element>`, so let's go with that.